src/services/messagesService.js

```diff
@@ -178,6 +178,12 @@ const getReactionsDetails = async function(token, messageId) {
 	return axios.get(generateOcsUrl('apps/spreed/api/v1/reaction/{token}/{messageId}', { token, messageId }))
 }
 
+const editMessage = async function({ token, messageId, updatedMessage }) {
+	return axios.put(generateOcsUrl('apps/spreed/api/v1/chat/{token}/{messageId}', { token, messageId }), {
+		message: updatedMessage,
+	})
```

Comment:
```suggestion
const editMessage = async function({ token, messageId, updatedMessage }, options) {
	return axios.put(generateOcsUrl('apps/spreed/api/v1/chat/{token}/{messageId}', { token, messageId }), {
		message: updatedMessage,
	}, options)
```
Could you please provide an options prop? It has to be added for all axios requests to be able to talk to remote servers